overlay/mobile-nixos/kernel/builder.nix

```diff
@@ -203,7 +203,7 @@ stdenv.mkDerivation (inputArgs // {
       sed -i scripts/ld-version.sh -e "s|/usr/bin/awk|${buildPackages.gawk}/bin/awk|"
     fi
   ''
-    + maybeString prePatch
+  + maybeString prePatch
```

Comment:
Since this is not touching the kernel builder anymore, drop this change.

(Re-formatting to be in-line with how everything else is done would be welcome if this file was changed otherwise.)